ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/datasource/metadata/tree/TreeNode.java

```diff
@@ -0,0 +1,11 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.metadata.tree;
+
+import javax.swing.tree.DefaultMutableTreeNode;
+
+public interface TreeNode {
+
+    public String getName();
+
+    public DefaultMutableTreeNode getMutableTreeNode();
```

Comment:
Access modifier is 'public' is redudant here.  In interfaces access modifier by default is public

Reply:
changed